sui/utils/cli-utils.js

```diff
@@ -44,6 +44,10 @@ const addBaseOptions = (program, options = {}) => {
         program.addOption(new Option('--txFilePath <file>', 'unsigned transaction will be stored'));
     }
 
+    if (options.allowCustomRpc) {
```

Comment:
Just nitpicking, but suggesting `options.customRpc` as it's a bit more succinct and the allow is impliled.